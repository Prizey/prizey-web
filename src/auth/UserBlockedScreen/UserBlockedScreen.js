import React from 'react'
import { Button, Typography } from '@material-ui/core'
import { withStyles } from '@material-ui/core/styles'
import Layout from 'design/Layout/Layout'

const styles = theme => ({
  pageTitle: {
    marginBottom: theme.spacing.sm,
  },
})

const createBlockedPage = withStyles(styles)(({ classes }) => (
  <Layout>
    <Typography align="center" variant="h2" className={classes.pageTitle}>
      You are blocked!
    </Typography>

    <Typography align="center" variant="h5">
      Due to recent events on your account history we blocked you, but do not
      worry, enter in contact through{' '}
      <Button
        href="mailto:contact@prizeyapp.com"
        variant="contained"
        color="primary"
        fullWidth
      >
        contact@prizeyapp.com
      </Button>
    </Typography>
  </Layout>
))

export default createBlockedPage
