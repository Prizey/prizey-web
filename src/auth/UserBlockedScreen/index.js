import UserBlockedScreen from './UserBlockedScreen'

export default UserBlockedScreen
