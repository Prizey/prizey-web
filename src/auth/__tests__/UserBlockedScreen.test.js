import React from 'react'
import renderer from 'react-test-renderer'
import UserBlockedScreen from '../UserBlockedScreen'

jest.mock('../../design/Layout/RegisterPageView')

it('renders correctly', () => {
  const tree = renderer.create(<UserBlockedScreen />).toJSON()
  expect(tree).toMatchSnapshot()
})
