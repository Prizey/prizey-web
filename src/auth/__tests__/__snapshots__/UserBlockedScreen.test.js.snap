// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Component-root-7"
>
  <div
    className="Component-header-5"
  />
  <div
    className="Component-container-4"
  >
    <img
      className="Component-logoImage-6"
      src="/logo.png"
    />
    <div
      className="Component-caption-2"
    />
    <div
      className="Component-children-3"
    >
      <h2
        className="MuiTypography-root-8 MuiTypography-h2-21 MuiTypography-alignCenter-31 Component-pageTitle-1"
      >
        You are blocked!
      </h2>
      <h5
        className="MuiTypography-root-8 MuiTypography-h5-24 MuiTypography-alignCenter-31"
      >
        Due to recent events on your account history we blocked you, but do not worry, enter in contact through
         
        <a
          className="MuiButtonBase-root-70 MuiButton-root-44 MuiButton-contained-55 MuiButton-containedPrimary-56 MuiButton-raised-58 MuiButton-raisedPrimary-59 MuiButton-fullWidth-69"
          href="mailto:contact@prizeyapp.com"
          onBlur={[Function]}
          onContextMenu={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex="0"
        >
          <span
            className="MuiButton-label-45"
          >
            contact@prizeyapp.com
          </span>
          <span
            className="MuiTouchRipple-root-73"
          />
        </a>
      </h5>
    </div>
    <p
      className="MuiTypography-root-8 MuiTypography-body1-17 MuiTypography-alignCenter-31"
    />
  </div>
</div>
`;
